Dockerfile

```diff
@@ -1,17 +1,36 @@
-FROM node:slim
+FROM node:21-slim as base
 
 EXPOSE 3000/tcp
 WORKDIR /usr/app
 COPY ./ ./
 
-SHELL ["/bin/bash", "-c"]
-
 RUN apt update && \
     apt install openssl -y && \
     apt clean && \
     apt autoclean && \
-    apt autoremove && \
-    npm install --ignore-scripts && \
-    npm install -g prisma
+    apt autoremove
+
+RUN npm ci --ignore-scripts
```

Comment:
I'd recommend doing everything in a single RUN directive not to add too many layers to the image.

Reply:
I also tried to get a version going where I just copy over the built files from the `build` image target to the `production` image target, but my lack of knowledge of how and what Next files i need in order to run it prohibited me from ending up with a slimmer image.

I might look at it again when I have more time, though.